spec/controllers/people_controller_spec.rb

```diff
@@ -0,0 +1,159 @@
+# of tools you can use to make these specs even more expressive, but we're
+# sticking to rails and rspec-rails APIs to keep things simple and stable.
+#
+# Compared to earlier versions of this generator, there is very limited use of
+# stubs and message expectations in this spec.  Stubs are only used when there
+# is no simpler way to get a handle on the object needed for the example.
+# Message expectations are only used when there is no simpler way to specify
+# that an instance is receiving a specific message.
+
+RSpec.describe PeopleController, type: :controller do
+
+  # This should return the minimal set of attributes required to create a valid
+  # Person. As you add validations to Person, be sure to
+  # adjust the attributes here as well.
+  let(:valid_attributes) {
+    skip("Add a hash of attributes valid for your model")
+  }
+
+  let(:invalid_attributes) {
+    skip("Add a hash of attributes invalid for your model")
+  }
+
+  # This should return the minimal set of values that should be in the session
+  # in order to pass any filters (e.g. authentication) defined in
+  # PeopleController. Be sure to keep this updated too.
+  let(:valid_session) { {} }
+
+  describe "GET #index" do
+    it "assigns all people as @people" do
+      person = Person.create! valid_attributes
+      get :index, {}, valid_session
+      expect(assigns(:people)).to eq([person])
+    end
+  end
+
+  describe "GET #show" do
+    it "assigns the requested person as @person" do
+      person = Person.create! valid_attributes
+      get :show, {:id => person.to_param}, valid_session
+      expect(assigns(:person)).to eq(person)
+    end
+  end
+
+  describe "GET #new" do
+    it "assigns a new person as @person" do
+      get :new, {}, valid_session
+      expect(assigns(:person)).to be_a_new(Person)
+    end
+  end
+
+  describe "GET #edit" do
+    it "assigns the requested person as @person" do
+      person = Person.create! valid_attributes
+      get :edit, {:id => person.to_param}, valid_session
+      expect(assigns(:person)).to eq(person)
+    end
+  end
+
+  describe "POST #create" do
+    context "with valid params" do
+      it "creates a new Person" do
+        expect {
+          post :create, {:person => valid_attributes}, valid_session
+        }.to change(Person, :count).by(1)
+      end
+
+      it "assigns a newly created person as @person" do
+        post :create, {:person => valid_attributes}, valid_session
+        expect(assigns(:person)).to be_a(Person)
+        expect(assigns(:person)).to be_persisted
+      end
+
+      it "redirects to the created person" do
+        post :create, {:person => valid_attributes}, valid_session
+        expect(response).to redirect_to(Person.last)
+      end
+    end
+
+    context "with invalid params" do
+      it "assigns a newly created but unsaved person as @person" do
+        post :create, {:person => invalid_attributes}, valid_session
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.